src/react/components/pages/export/exportForm.test.tsx

```diff
@@ -3,20 +3,37 @@ import ExportForm, { IExportFormProps, IExportFormState } from "./exportForm";
 import { mount } from "enzyme";
 import { IExportFormat } from "../../../../models/applicationState";
 import { ExportAssetState } from "../../../../providers/export/exportProvider";
+import MockFactory from "../../../../common/mockFactory";
+
+jest.mock("../../../../providers/export/exportProviderFactory");
+import { ExportProviderFactory } from "../../../../providers/export/exportProviderFactory";
 
 describe("Export Form Component", () => {
+    const exportProviderRegistrations = MockFactory.createExportProviderRegistrations();
+
     function createComponent(props: IExportFormProps) {
         return mount(
             <ExportForm {...props} />,
         );
     }
 
+    beforeAll(() => {
+        Object.defineProperty(ExportProviderFactory, "handlers", {
+            get: jest.fn(() => exportProviderRegistrations),
+        });
+    });
+
     const onSubmitHandler = jest.fn();
 
     it("State is initialized without export settings", () => {
         const defaultExportType = "vottJson";
         const props: IExportFormProps = {
-            settings: null,
+            settings: {
```

Comment:
FYI - this test specifically is testing the scenario when the settings have been been previously saved, so they should be null/undefined.

Reply:
Will update this with my final fix tests PR coming later today